utonics/labproject/main.go

```diff
@@ -242,21 +242,44 @@ func newProject(values map[string][]string, botClient, userClient *worker.Client
 		fmt.Printf(string(stdout))
 		fmt.Printf(string(stderr))
 	}
-	submoduleForEach("git", "checkout", "master")
+	submoduleForEach("git", "checkout", "master") // TODO: find default branch instead
 	submoduleForEach("git", "pull")
-
+	//submoduleForEach("gin", "init") // TODO: make this work !
```

Comment:
Is this meant for the current PR?  If not, maybe it's best to open an issue than leave TODOs in code.

Reply:
Planning an issue after the PR, or linking to this PR to get it solved before the merge (but I do not know who could do that apart from you :) )